lxd/cluster/config.go

```diff
@@ -265,6 +275,17 @@ func offlineThresholdValidator(value string) error {
 	return nil
 }
 
+func imageSyncNodeCountValidator(value string) error {
+	imageSyncNodeCount, err := strconv.Atoi(value)
+	if err != nil {
+		return fmt.Errorf("image sync node count is not a number")
```

Comment:
All errors must start with a capital letter (yes, I realize the function above this one does it wrong...)